File: app/src/main/java/com/bannersimple/MainActivity.java
package com.bannersimple;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.bannersimple.simple.ImageManagerActivity;
import com.bannersimple.simple.MethodTestActivity;
import com.bannersimple.simple.RefreshActivity;
import com.bannersimple.simple.SimpleActivity;
import com.bannersimple.simple.TransformerActivity;
import com.bannersimple.simple.issues.Issues10Activity;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btn_refresh_simple).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(v.getContext(), RefreshActivity.class);
                intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                startActivity(intent);
            }
        });

        findViewById(R.id.btn_simple).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(v.getContext(), SimpleActivity.class);
                intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                startActivity(intent);
            }
        });

        findViewById(R.id.btn_image_manager).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(v.getContext(), ImageManagerActivity.class);
                intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                startActivity(intent);
            }
        });

        findViewById(R.id.btn_transformer).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(v.getContext(), TransformerActivity.class);
                intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                startActivity(intent);
            }
        });

        findViewById(R.id.btn_method_test).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(v.getContext(), MethodTestActivity.class);
                intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                startActivity(intent);
            }
        });

        findViewById(R.id.btn_issues_10).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(v.getContext(), Issues10Activity.class);
                intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                startActivity(intent);
            }
        });
    }

}

File: app/src/main/java/com/bannersimple/MainActivity.kt
package com.bannersimple

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.bannersimple.simple.*
import com.bannersimple.simple.issues.Issues10Activity
import com.bannersimple.simple.issues.Issues12Activity
import com.bannersimple.simple.issues.Issues13Activity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_refresh_simple.setOnClickListener { v ->
            val intent = Intent(v.context, RefreshActivity::class.java)
            startActivity(intent)
        }

        btn_simple.setOnClickListener { v ->
            val intent = Intent(v.context, SimpleActivity::class.java)
            startActivity(intent)
        }

        btn_guide.setOnClickListener { v ->
            val intent = Intent(v.context, SimpleGuideActivity::class.java)
            startActivity(intent)
        }

        btn_image_manager.setOnClickListener { v ->
            val intent = Intent(v.context, ImageManagerActivity::class.java)
            startActivity(intent)
        }

        btn_transformer.setOnClickListener { v ->
            val intent = Intent(v.context, TransformerActivity::class.java)
            startActivity(intent)
        }

        btn_method_test.setOnClickListener { v ->
            val intent = Intent(v.context, MethodTestActivity::class.java)
            startActivity(intent)
        }

        btn_issues_10.setOnClickListener { v ->
            val intent = Intent(v.context, Issues10Activity::class.java)
            startActivity(intent)
        }

        btn_issues_12.setOnClickListener { v ->
            val intent = Intent(v.context, Issues12Activity::class.java)
            startActivity(intent)
        }

        btn_issues_13.setOnClickListener { v ->
            val intent = Intent(v.context, Issues13Activity::class.java)
            startActivity(intent)
        }

        btn_java.setOnClickListener { v ->
            val intent = Intent(v.context, SimpleJavaActivity::class.java)
            startActivity(intent)
        }
    }

}
